Source/WebKit/UIProcess/WebProcessPool.h

```diff
@@ -447,6 +447,10 @@ class WebProcessPool final
     void lockdownModeStateChanged();
 #endif
 
+#if ENABLE(WEBCONTENT_CRASH_TESTING)
+    bool crashWhenCreatingWebProcess() const { return m_crashWhenCreatingWebProcess; }
```

Comment:
I think both the variable and function name should start with "should" for clarity. The current naming is a little bit confusing in my opinion.

Reply:
I'll make this change.